src/broker.js

```diff
@@ -150,11 +150,12 @@ module.exports = async (config) => {
         }
 
         if (config.reporting.perNodeMetrics && config.reporting.perNodeMetrics.enabled) {
-            // streamIds.secStreamId = await createMetricsStream('/streamr/node/metrics/sec')
-            // streamIds.minStreamId = await createMetricsStream('/streamr/node/metrics/min')
-            // streamIds.hourStreamId = await createMetricsStream('/streamr/node/metrics/hour')
-            // streamIds.dayStreamId = await createMetricsStream('/streamr/node/metrics/day')
-            logger.info('Starting perNodeMetrics -- Not implemented yet')
+            // await client.ensureConnected()
+            streamIds.secStreamId = await createMetricsStream('/streamr/node/metrics/sec')
```

Comment:
Hmm streams are getting created now both here and in the class `StreamMetrics`. It would be probably more clear to do them only in one of these.

Reply:
I.e. you can probably remove these here now?